api/MediaStreamTrack.json

```
@@ -1119,6 +1119,40 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "screenPixelRatio": {
+          "__compat": {
+            "description": "Return object contains `screenPixelRatio`",
```

To be consistent with `RTCRtp{Receiver,Sender}.getProperties()`:

```suggestion
        "return_object_property_screenPixelRatio": {
          "__compat": {
            "description": "`screenPixelRatio` property in returned object",
```

---

> Yes, I wondered if there was (or should be) some standardised & documented way for a subfeature to say "returned object contains property x". Also wonder if it would be helpful to link to the page for the property.

@wbamberg yup, this is a good idea. I'm going to add this.

---

@caugner link added.